src/Controls/src/Core/Handlers/Shell/ShellHandler.Windows.cs

```diff
@@ -124,6 +127,20 @@ public static void MapFlyoutBackground(ShellHandler handler, Shell view)
 					view.FlyoutBackgroundColor?.AsPaint());
 		}
 
+		public static void MapFlyoutIcon(ShellHandler handler, Shell view)
+		{
+			var togglePaneButton = handler.PlatformView.TogglePaneButton;
+
+			if (togglePaneButton is null)
+				return;
+
+			var icon = togglePaneButton.GetFirstDescendant<AnimatedIcon>();
+			var flyoutIcon = view.FlyoutIcon;
+			var provider = handler.GetRequiredService<IImageSourceServiceProvider>();
+
+			icon.UpdateFlyoutIconAsync(flyoutIcon, provider).FireAndForget();
```

Comment:
Can we make the extension method against `MauiNavigationView` and then `UpdateFlyoutIcon` will just be against `MauiNavigationView` ? I think that'll make it easier to reuse if folks want to reuse the extension method.

Reply:
Yeah, would be easier to extend. Added.